openstack/dcaas/v2/dc-endpoint-group/Create.go

```diff
@@ -0,0 +1,31 @@
+package dc_endpoint_group
+
+import (
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/build"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+)
+
+type CreateOpts struct {
```

Comment:
Missing required `tenant_id` parameter:
https://docs.otc.t-systems.com/direct-connect/api-ref/apis/direct_connect_endpoint_group/creating_an_endpoint_group.html#en-dc-topic-0055025336
May be documentation bug, please comment